lib/json-parser.js

```diff
@@ -149,14 +149,14 @@ function compareLocations(span, msg) {
     && span.column_end === msg.col_end;
 }
 
-// Appends spans's lable to the main message. It only adds the lable if:
+// Appends spans's label to the main message. It only adds the label if:
 // - the main message doesn't contain exactly the same phrase
 // - the main message doesn't contain the same information but uses different wording
 function appendSpanLabel(span, msg) {
   if (!span.label || msg.message.indexOf(span.label) >= 0) {
     return;
   }
-  for (let idx = 0; idx < redundantLabels.length; idx++) {
+  for (idx in redundantLabels) {
```

Comment:
It's not only `var`, Travis requires an additional `if` within the loop because it thinks that I iterate through the simple object properties and must check if those properties are internal or not. With such check the loop becomes more wordy. It seems the original loop was more concise than any other option.